k8s/storage/storageclass.go

```diff
@@ -46,6 +50,28 @@ func (c *Client) GetStorageClass(name string) (*storagev1.StorageClass, error) {
 	return c.storage.StorageClasses().Get(context.TODO(), name, metav1.GetOptions{})
 }
 
+// GetDefaultStorageClasses returns all storageClasses that are set as default
+func (c *Client) GetDefaultStorageClasses() (*storagev1.StorageClassList, error) {
+        defaultStorageClasses := &storagev1.StorageClassList{}
+        var storageClassList *storagev1.StorageClassList
+        if err := c.initClient(); err != nil {
+                return nil, err
+        }
+
+        storageClassList, err := c.storage.StorageClasses().List(context.TODO(), metav1.ListOptions{})
+        if err != nil {
+                return nil, err
+        }
+        for _, sc := range storageClassList.Items {
+                if val, ok := sc.Annotations[defaultStorageclassAnnotationKey]; ok {
```

Comment:
So here if they have annotations set on different storage classes, then we would gibe out the first one from the is it? 
Everytime this API is called then there is no guarantee it would be same SC in case of multiple default SC is set?

Reply:
Yes, sched-ops will return all the storageclass that are set to default. If more than one sc is set as default, it is up to callers to handle the return value.
In the case of stork, I am handling both the number of default sc greater than one or zero.